services/gateway/tests/constants/registerApi.js

```diff
@@ -171,8 +171,8 @@ const expectedResponseForRegisterRpcApi = {
 				'get.pos.validators': 'indexer.pos.validators',
 				'get.token.account.exists': 'indexer.token.account.exists',
 				'get.token.constants': 'indexer.token.constants',
-				'get.tokens': 'indexer.tokens',
-				'get.tokens.summary': 'indexer.tokens.summary',
+				'get.token.balances': 'indexer.tokens',
```

Comment:
Could we also rename the methods within indexer microservice from `tokens` to `token.balances`? Please leave the method names in connector as is.